test/unit/App/middlewares/arguments.spec.ts

```diff
@@ -761,5 +762,92 @@ describe('App middleware and listener arguments', () => {
 
       sinon.assert.calledOnce(fakeAck);
     });
+
+    it('should have function executed event details from a custom step payload', async () => {
+      const fakeAxiosPost = sinon.fake.resolves({});
+      overrides = buildOverrides([withNoopWebClient(), withAxiosPost(fakeAxiosPost)]);
+      const MockApp = await importApp(overrides);
+      const callbackId = 'reverse_string';
+      const functionBotAccessToken = 'xwfp-example';
+      const functionExecutionId = 'Fx1234567890';
+      const inputs = {
+        numerics: 12,
+        stringToReverse: 'palindrome',
+        trimmedSpaces: false,
+      };
+
+      const app = new MockApp({
+        attachFunctionToken: false, // https://github.com/slackapi/bolt-js/pull/2513
+        receiver: fakeReceiver,
+        authorize: sinon.fake.resolves(dummyAuthorizationResult),
+      });
+
+      let called = false;
+      app.function(callbackId, async ({ context }) => {
+        assert.strictEqual(context.functionExecutionId, functionExecutionId);
+        assert.strictEqual(context.functionBotAccessToken, functionBotAccessToken);
```

Comment:
```suggestion
        assert.strictEqual(context.functionExecutionId, functionExecutionId);
        assert.strictEqual(context.functionBotAccessToken, functionBotAccessToken);
        assert.deepStrictEqual(context.functionInputs, inputs);
```

📣 Here we are **not** testing that `functionInputs` are added to `context` but I'm not sure if this is something we want to change in this PR?

It's not clear to me if this should be included following `L1624` from `body.event.inputs` for convenience:

https://github.com/slackapi/bolt-js/blob/76140ed83bf8e2b7aa2db37c12ec8ae779be98f5/src/App.ts#L1616-L1635

Reply:
Good thinking. I'm finding this change involves an update to another file which does seem outside the scope of this PR 🫡